src/components/cards/Node.tsx

```diff
@@ -16,11 +17,14 @@ export function NodeCard({ data }: any) {
       <Handle type="target" position={Position.Top} id={data.Key} />
       <Link href={data?.Url} target="_blank">
         <div
-          className={`py-4 px-2 text-xs w-52 rounded-md border-solid border-2 hover:border-4 box-content hover:relative hover:border-purple-500 ${bgColor}`}
+          className={`py-4 px-2 text-xs w-52 rounded-md border-solid border-2 hover:border-4 box-content hover:relative transition-colors duration-1000hover:border-purple-500 ${bgColor} ${
+            highlighted && "border-4 border-[#7dcff5]"
+          }`}
```

Comment:
@kareraolivier  can you style border of highlighted nodes  using that bluish color which fade away  like it is on design.

Reply:
done